cmake/templates/BoostConfig.cmake.in

```diff
@@ -14,6 +14,24 @@ else()
   endfunction()
 endif()
 
+function(hunter_add_rt_library target_name)
+  include(CheckCXXSymbolExists)
+  check_cxx_symbol_exists("clock_gettime" "time.h" HAVE_CLOCK_GETTIME)
```

Comment:
`HAVE_CLOCK_GETTIME` variable will go to cache, let's prefix the name with `_hunter`.